Helpers/MediaLaunchObject.h

```diff
@@ -0,0 +1,36 @@
+//
+//  MediaLaunchObject.h
+//  ConnectSDK
+//
+//  Created by Ibrahim Adnan on 1/19/15.
+//  Copyright (c) 2015 LG Electronics. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "MediaControl.h"
+#import "PlayListControl.h"
+#import "LaunchSession.h"
+
+@interface MediaLaunchObject : NSObject
+
+/*! MediaControl object of Media player*/
+@property (nonatomic, strong) id<MediaControl> mediaControl;
+
+/*! PlayList Control Object of Media player*/
+@property (nonatomic, strong) id<PlayListControl> playListControl;
+
+/*! Launch Session object of Media player*/
+@property (nonatomic, strong) LaunchSession *session;
+
+
+/*
```

Comment:
You should use `/** */`-style comment for documentations. `/* */` are just generic comments.